src/instructlab/configuration.py

```diff
@@ -333,6 +333,10 @@ class _evaluate(BaseModel):
     gpus: Optional[int] = Field(
         default=None, description="Number of GPUs to use for running evaluation."
     )
+    device: str = Field(
+        default="cpu",
+        description="PyTorch device (e.g. `cpu`, `cuda`, or `mps`) for running models."
```

Comment:
Please use a `pattern="cpu|cuda|mps"` to get validation.

Reply:
FYI -- I've used device=xpu to support Intel GPUs for another project.  I'm guessing you'd be smart to stick with the current list and add as-needed, but thought I'd throw that out there.

Reply:
Thanks @markstur - I'll keep with this list for now, but the idea is to add as we go. I will keep xpu in mind